core/capabilities/remote/executable/server.go

```diff
@@ -203,6 +203,11 @@ func (r *server) getMessageHash(msg *types.MessageBody) ([32]byte, error) {
 		return [32]byte{}, fmt.Errorf("failed to unmarshal capability request: %w", err)
 	}
 
+	// default to excluding StepDependency
+	if len(r.config.RequestHashExcludedAttributes) == 0 {
+		r.config.RequestHashExcludedAttributes = []string{"StepDependency"}
```

Comment:
Can we double check in the logs for staging that this is the right structure of the message? If this is wrong you'll have to do a back and forth and redeploy to validate the fix